scripts/docs.js

```diff
@@ -0,0 +1,19 @@
+let searchLinks = document.querySelector(".search-links");
+let search = document.querySelector(".search");
+// let html = document.querySelector("html");
+
+const showSearchLinks = () => {
+    searchLinks.classList.remove("hide");
+}
+
+// const hideSearchLinks = (e) => {
+//     if(e.target != searchLinks){
+//     searchLinks.classList.add("hide");}
+//     else{
+//         searchLinks.classList.remove("hide");
+//     }
+// }
+
+// document.addEventListener("click",hideSearchLinks);
+
```

Comment:
You should remove these comments if not required

Reply:
I am thinking of editing the commented code for future functionalities. That's why I didn't remove it